src/k8s.rs

```diff
@@ -175,8 +176,11 @@ pub struct Context {
     impersonate_user: Option<String>,
     connect_timeout_secs: u32,
     read_timeout_secs: u32,
+    custom_dns_mapping: Option<(String, IpAddr)>,
+    tls_server_name: Option<String>,
```

Comment:
This doesn't seem to actually be used. You pass it around a bunch, and it seemed you wanted to use it in `get_client`, but for now you're not so maybe we can just remove it?